cmd/root.go

```diff
@@ -31,6 +33,80 @@ var rootCmd = &cobra.Command{
 	},
 }
 
+var componentUpdateCmd = &cobra.Command{
+	Use: "update",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 2 {
+			// TODO: Improve this message
+			return fmt.Errorf("Invalid arguments, maximum 1 arguments possible")
+		}
+		return nil
+	},
+	Short: "update the component",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+
+		if len(componentBinary) != 0 {
+			fmt.Printf("--binary is not implemented yet\n\n")
+			cmd.Help()
+			os.Exit(1)
+		}
+
+		var (
+			componentName string
+			err           error
+		)
+		if len(args) == 0 {
+			componentName, err = component.GetCurrent(client)
+			if err != nil {
+				fmt.Println("Unable to get current component")
+			}
+		} else {
+			componentName = args[0]
+		}
+
+		if len(componentBinary) != 0 {
+			fmt.Printf("--binary is not implemented yet\n\n")
+			os.Exit(1)
+		}
+
```

Comment:
there should be one more check to check that only one flag has been set (only `--git` or `--local` or `--binary`)